draft-ietf-quic-multipath.md

```diff
@@ -744,26 +744,26 @@ second path, the server's 1-RTT packets use DCID C2, which has a sequence
 number of 2; the client's 1-RTT packets use DCID S3, which has a sequence number
 of 3. Note that the paths use different packet number spaces. In this case, the
 client is going to close the first path. It identifies the path by the sequence
-number of the received packet's DCID over that path,
-hence using the path_id 1. Optionally, the server confirms the path closure
+number of the DCID used for sending packets over that path,
+hence using the path_id 2. Optionally, the server confirms the path closure
 by sending an PATH_ABANDON frame using
-the sequence number of the received packet's DCID over that path as path
-identifier, which corresponds to the path_id 2. Both the client and
+the sequence number of the DCID used to send over that path as path
+identifier, which corresponds to the path_id 1. Both the client and
 the server can close the path after receiving the RETIRE_CONNECTION_ID frame
 for that path.
 
 ~~~
 Client                                                      Server
 
 (client tells server to abandon a path)
-1-RTT[X]: DCID=S2 PATH_ABANDON[path_id=1]->
+1-RTT[X]: DCID=S2 PATH_ABANDON[path_id=2]->
                            (server tells client to abandon a path)
-                      <-1-RTT[Y]: DCID=C1 PATH_ABANDON[path_id=2],
+                      <-1-RTT[Y]: DCID=C1 PATH_ABANDON[path_id=1],
                                                ACK_MP[Seq=2, PN=X]
 (client retires the corresponding CID)
-1-RTT[U]: DCID=S3 RETIRE_CONNECTION_ID[2], ACK_MP[Seq=1, PN=Y] ->
+1-RTT[U]: DCID=S3 RETIRE_CONNECTION_ID[1], ACK_MP[Seq=1, PN=Y] ->
                             (server retires the corresponding CID)
- <- 1-RTT[V]: DCID=C2 RETIRE_CONNECTION_ID[1], ACK_MP[Seq=3, PN=U]
+ <- 1-RTT[V]: DCID=C2 RETIRE_CONNECTION_ID[2], ACK_MP[Seq=3, PN=U]
```

Comment:
```suggestion
 <- 1-RTT[V]: DCID=C2 RETIRE_CONNECTION_ID[1], ACK_MP[Seq=3, PN=U]
```
Server wants to retire DCID C1, so the sequence number field in the retire_connection_ID should be 1.